### DISCUSSION TITLE
Using "Sprite Previewer"

### DISCUSSION THREAD

**Discussion by Nikv2005** [Monika-After-Story/MonikaModDev#9183] [QA - code and content creation (technical)]
Is it save to use Sprite Previewer on main version on game? Doesn't that be dangerous?  because i don't want to work with persistent file when this thing broke something. And as i understand if i want to delete it i can just delete this rpy file. Just a question , i want to make everything save for sure.

**Comment by Wingdinggaster656** [comment]
Sprite Preview is 100% safe.

The only problem you might hate is that after selecting this topic, your persistent files remember that you once chose it. Nothing else.
So the only problem is that there is a record of the test topic in the persistent files.

**Comment by Nikv2005** [comment]
> Sprite Preview is 100% safe.
> 
> The only problem you might hate is that after selecting this topic, your persistent files remember that you once chose it. Nothing else.
> So the only problem is that there is a record of the test topic in the persistent files.

How will it affect? I mean there differences , but which differences? There is possible to just download this thing , get needed emotions and after that replace persistent on which doesn't remember this thing?

**Comment by Wingdinggaster656** [comment]
> > Sprite Preview is 100% safe.
> > The only problem you might hate is that after selecting this topic, your persistent files remember that you once chose it. Nothing else.
> > So the only problem is that there is a record of the test topic in the persistent files.
> 
> How will it affect? I mean there differences , but which differences? There is possible to just download this thing , get needed emotions and after that replace persistent on which doesn't remember this thing?

You can.
And actually just leave that topic memory there is nothing wrong.
But well, maybe it sounds bad to you.

**Comment by Booplicate** [comment]
Don't use dev tools in your main install, create a separate install with local persistent and use it for testing.

**Comment by Nikv2005** [comment]
> Don't use dev tools in your main install, create a separate install with local persistent and use it for testing.

This is possible to make a local persistent? Because persistent file in appdata and i can't imagine how i can create local file... Maybe there is possible delete tool and then  replace persistent file after work?

**Comment by Wingdinggaster656** [comment]
> > Don't use dev tools in your main install, create a separate install with local persistent and use it for testing.
> 
> This is possible to make a local persistent? Because persistent file in appdata and i can't imagine how i can create local file... Maybe there is possible delete tool and then replace persistent file after work?

You can replace. As long as you think this is fine with you.

However you could also try to create a local file. See our wiki.

**Comment by Booplicate** [comment]
@Nikv2005 [this](https://github.com/Monika-After-Story/MonikaModDev/wiki/Contributing-Guidelines#bug-fixes-and-new-software-features) tells you how to make persistent local to an install.

**Comment by Nikv2005** [comment]
> @Nikv2005 [this](https://github.com/Monika-After-Story/MonikaModDev/wiki/Contributing-Guidelines#bug-fixes-and-new-software-features) tells you how to make persistent local to an install.

Yeah i see , thanks.
